diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1aa7839..785b90d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -258,27 +258,27 @@ set(_INSTALL_DESTINATIONS
 ###
 ### Library
 ###
-add_library(yaml-cpp ${library_sources})
+add_library(yaml-cpp0.6 ${library_sources})
 
 if (NOT CMAKE_VERSION VERSION_LESS 2.8.12)
-    target_include_directories(yaml-cpp
+    target_include_directories(yaml-cpp0.6
         PUBLIC $<BUILD_INTERFACE:${YAML_CPP_SOURCE_DIR}/include>
                $<INSTALL_INTERFACE:${INCLUDE_INSTALL_ROOT_DIR}>
         PRIVATE $<BUILD_INTERFACE:${YAML_CPP_SOURCE_DIR}/src>)
 endif()
 
-set_target_properties(yaml-cpp PROPERTIES
+set_target_properties(yaml-cpp0.6 PROPERTIES
   COMPILE_FLAGS "${yaml_c_flags} ${yaml_cxx_flags}"
 )
 
-set_target_properties(yaml-cpp PROPERTIES
+set_target_properties(yaml-cpp0.6 PROPERTIES
 	VERSION "${YAML_CPP_VERSION}"
 	SOVERSION "${YAML_CPP_VERSION_MAJOR}.${YAML_CPP_VERSION_MINOR}"
-	PROJECT_LABEL "yaml-cpp ${LABEL_SUFFIX}"
+	PROJECT_LABEL "yaml-cpp0.6 ${LABEL_SUFFIX}"
 )
 
 if(IPHONE)
-	set_target_properties(yaml-cpp PROPERTIES
+	set_target_properties(yaml-cpp0.6 PROPERTIES
 		XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET "3.0"
 	)
 endif()
@@ -286,7 +286,7 @@ endif()
 if(MSVC)
 	if(NOT BUILD_SHARED_LIBS)
 		# correct library names
-		set_target_properties(yaml-cpp PROPERTIES
+		set_target_properties(yaml-cpp0.6 PROPERTIES
 			DEBUG_POSTFIX "${LIB_TARGET_SUFFIX}d"
 			RELEASE_POSTFIX "${LIB_TARGET_SUFFIX}"
 			MINSIZEREL_POSTFIX "${LIB_TARGET_SUFFIX}"
@@ -296,7 +296,7 @@ if(MSVC)
 endif()
 
 if (YAML_CPP_INSTALL)
-	install(TARGETS yaml-cpp EXPORT yaml-cpp-targets ${_INSTALL_DESTINATIONS})
+	install(TARGETS yaml-cpp0.6 EXPORT yaml-cpp0.6-targets ${_INSTALL_DESTINATIONS})
 	install(
 		DIRECTORY ${header_directory}
 		DESTINATION ${INCLUDE_INSTALL_DIR}
@@ -305,14 +305,14 @@ if (YAML_CPP_INSTALL)
 endif()
 
 export(
-    TARGETS yaml-cpp
-    FILE "${PROJECT_BINARY_DIR}/yaml-cpp-targets.cmake")
-export(PACKAGE yaml-cpp)
-set(EXPORT_TARGETS yaml-cpp CACHE INTERNAL "export targets")
+    TARGETS yaml-cpp0.6
+    FILE "${PROJECT_BINARY_DIR}/yaml-cpp0.6-targets.cmake")
+export(PACKAGE yaml-cpp0.6)
+set(EXPORT_TARGETS yaml-cpp0.6 CACHE INTERNAL "export targets")
 
 set(CONFIG_INCLUDE_DIRS "${YAML_CPP_SOURCE_DIR}/include")
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/yaml-cpp-config.cmake.in
-	"${PROJECT_BINARY_DIR}/yaml-cpp-config.cmake" @ONLY)
+	"${PROJECT_BINARY_DIR}/yaml-cpp0.6-config.cmake" @ONLY)
 
 if(WIN32 AND NOT CYGWIN)
 	set(INSTALL_CMAKE_DIR CMake)
@@ -324,20 +324,20 @@ endif()
 file(RELATIVE_PATH REL_INCLUDE_DIR "${CMAKE_INSTALL_PREFIX}/${INSTALL_CMAKE_DIR}" "${CMAKE_INSTALL_PREFIX}/${INCLUDE_INSTALL_ROOT_DIR}")
 set(CONFIG_INCLUDE_DIRS "\${YAML_CPP_CMAKE_DIR}/${REL_INCLUDE_DIR}")
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/yaml-cpp-config.cmake.in
-	"${PROJECT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/yaml-cpp-config.cmake" @ONLY)
+	"${PROJECT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/yaml-cpp0.6-config.cmake" @ONLY)
 
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/yaml-cpp-config-version.cmake.in
-	"${PROJECT_BINARY_DIR}/yaml-cpp-config-version.cmake" @ONLY)
+	"${PROJECT_BINARY_DIR}/yaml-cpp0.6-config-version.cmake" @ONLY)
 
 if (YAML_CPP_INSTALL)
 	install(FILES
-		"${PROJECT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/yaml-cpp-config.cmake"
-		"${PROJECT_BINARY_DIR}/yaml-cpp-config-version.cmake"
+		"${PROJECT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/yaml-cpp0.6-config.cmake"
+		"${PROJECT_BINARY_DIR}/yaml-cpp0.6-config-version.cmake"
 		DESTINATION "${INSTALL_CMAKE_DIR}" COMPONENT dev)
-	install(EXPORT yaml-cpp-targets DESTINATION ${INSTALL_CMAKE_DIR})
+	install(EXPORT yaml-cpp0.6-targets DESTINATION ${INSTALL_CMAKE_DIR})
 
 	if(UNIX)
-		set(PC_FILE ${CMAKE_BINARY_DIR}/yaml-cpp.pc)
+		set(PC_FILE ${CMAKE_BINARY_DIR}/yaml-cpp0.6.pc)
 		configure_file("yaml-cpp.pc.cmake" ${PC_FILE} @ONLY)
 		install(FILES ${PC_FILE} DESTINATION ${LIB_INSTALL_DIR}/pkgconfig)
 	endif()
diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index 8bdf303..16db918 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -57,7 +57,7 @@ set_target_properties(run-tests PROPERTIES
     COMPILE_FLAGS "${yaml_c_flags} ${yaml_cxx_flags} ${yaml_test_flags}"
 )
 target_link_libraries(run-tests
-    yaml-cpp
+    yaml-cpp0.6
     gmock
     ${CMAKE_THREAD_LIBS_INIT})
 
diff --git a/yaml-cpp.pc.cmake b/yaml-cpp.pc.cmake
index 3db7962..5170110 100644
--- a/yaml-cpp.pc.cmake
+++ b/yaml-cpp.pc.cmake
@@ -3,9 +3,9 @@ exec_prefix=${prefix}
 includedir=${prefix}/@INCLUDE_INSTALL_ROOT_DIR@
 libdir=${exec_prefix}/@LIB_INSTALL_DIR@
 
-Name: Yaml-cpp
+Name: Yaml-cpp0.6
 Description: A YAML parser and emitter for C++
 Version: @YAML_CPP_VERSION@
 Requires:
-Libs: -L${libdir} -lyaml-cpp
+Libs: -L${libdir} -lyaml-cpp0.6
 Cflags: -I${includedir}
